### ISSUE TITLE
YouTube Playlist Downloader cannot be accessed using a screen reader

### ISSUE THREAD

**Issue by MesterPerfect** [shaked6540/YoutubePlaylistDownloader#165]
hi 
I am an NVDA screen reader user
The NVDA screen reader cannot read the names of the buttons on the program's main interface
While he can read all the items in the settings without a problem
Can you fix that

**Comment by shaked6540** [comment]
Hey. I am sorry but I don't know what a screen reader is or how it works. If you could provide me with some information I could try to modify the program to work with it.

**Comment by MesterPerfect** [comment]
> Hey. I am sorry but I don't know what a screen reader is or how it works. If you could provide me with some information I could try to modify the program to work with it.


    You can view this guide
    https://learn.microsoft.com/en-us/windows/apps/design/accessibility/basic-accessibility-information#labels

**Comment by MesterPerfect** [comment]
Good morning
Did you come up with something regarding accessibility support?

**Comment by shaked6540** [comment]
Not yet. Its exam period right now so I dont have time to do anything other than studying. I'll get to it at the end of the month

**Comment by MesterPerfect** [comment]
ok, good luck on exam
